crates/rover-client/build.rs

```diff
@@ -65,12 +67,38 @@ fn main() -> std::io::Result<()> {
     }
 }
 
-fn update_schema(client: &Client, url: &str) -> std::io::Result<()> {
+fn retry_request(client: &Client, method: Method, url: &str) -> Result<Response> {
+    let mut delay_secs = 2;
+    let max_delay_secs = 30;
+    let mut has_failed = false;
+
+    loop {
```

Comment:
rather than manually reimplementing this retry logic, might just want to use the `retry` crate as per here: https://github.com/getditto/ditto/blob/develop/hydra/hydra-common/src/transaction_log.rs#L76. we've already incurred the dependency via hydra-common.